homeassistant/bootstrap.py

```diff
@@ -332,15 +332,17 @@ def async_enable_logging(
         not err_path_exists and os.access(err_dir, os.W_OK)
     ):
 
+        err_handler: logging.FileHandler
         if log_rotate_days:
-            err_handler: logging.FileHandler = (
-                logging.handlers.TimedRotatingFileHandler(
-                    err_log_path, when="midnight", backupCount=log_rotate_days
-                )
+            err_handler = logging.handlers.TimedRotatingFileHandler(
+                err_log_path, when="midnight", backupCount=log_rotate_days, delay=True
```

Comment:
We had `delay=True` initially only on the normal logger so people could have a look at the log before it was overwritten. Now that we always store the last log, we can drop the `delay=True` and shouldn't add it here either. That way `home-assistant.log` = log of current process.

Reply:
Thanks for the explanation, removed from both.